chain/sync_test.go

```diff
@@ -45,6 +45,11 @@ func init() {
 	policy.SetSupportedProofTypes(abi.RegisteredSealProof_StackedDrg2KiBV1)
 	policy.SetConsensusMinerMinPower(abi.NewStoragePower(2048))
 	policy.SetMinVerifiedDealSize(abi.NewStoragePower(256))
+
+	// these tests assume really fast block times. disabling
+	// the consistent broadcast delay to avoid them from adding
+	// an unnecessary overhead.
+	build.CBDeliveryDelay = 2 * time.Millisecond
```

Comment:
You mean deferring setting back this to the original value at the end of the last test (assuming that tests run sequentially), or at the end of every test in case they are run randomly?